docker/init.sh

```diff
@@ -2,9 +2,6 @@
 
 set -e
 
-script="$0"
-DIR="$(dirname $script)"
```

Comment:
Was this removal intended here? If so can you provide more context please?

Reply:
Yes. The `$DIR` variable was created to find the directory the current script was running on. I then used it to run another script within the same directory in the created DBUS session. I forgot to remove it once I removed the additional script.